stdlib/2/urllib2.pyi

```diff
@@ -22,7 +33,7 @@ class Request(object):
     unverifiable = ...  # type: bool
     type = ...  # type: Optional[str]
     origin_req_host = ...
-    unredirected_hdrs = ...
+    unredirected_hdrs: Dict[str, str] = ...
```

Comment:
Can remove the `= ...`. (I'm guessing this also supports unicode keys and values, but typing that correctly is hard.)

Reply:
Done in 9fea4210f9b8d83812cf4c54ee8655c4a64c264f